rendermodules/utilities/uri_utils.py

```diff
@@ -0,0 +1,89 @@
+from uri_handler.utils.uri_utils import (
+     uri_join,
+     get_prefix_path,
+     uri_prefix)
+
+from uri_handler.uri_functions import (
+    uri_readbytes,
+    uri_writebytes)
+
+__all__ = [
+    "uri_join", "uri_prefix", "uri_readbytes", "uri_writebytes"]
+
+# def join_paths(paths, delimiter='/'):

```

Comment:
are you leaving all these commented functions here intentionally?

Reply:
I was not -- deleted now.